tests/integration/nets_test1/test_podman_compose_nets_test1.py

```diff
@@ -59,9 +59,13 @@ def test_nets_test1(self):
             )
 
             # check if Host port is the same as provided by the service port
+            self.assertIsNotNone(container_info['NetworkSettings']["Ports"].get("8001/tcp", None))
+            self.assertGreater(len(container_info['NetworkSettings']["Ports"]["8001/tcp"]), 0)
+            self.assertIsNotNone(
+                container_info['NetworkSettings']["Ports"]["8001/tcp"][0].get("HostPort", None)
+            )
             self.assertEqual(
-                container_info['NetworkSettings']["Ports"],
-                {"8001/tcp": [{"HostIp": "", "HostPort": "8001"}]},
+                container_info['NetworkSettings']["Ports"]["8001/tcp"][0]["HostPort"], "8001"
```

Comment:
Why this change? Did the test start failing? The new version checks less things with more code.

Reply:
The original code's intention is just to validate the port as inidated by the comments:

> # check if Host port is the same as provided by the service port

The the test did fail on my laptop w/ podman 5.4.1. So I believe just focus on verifying service port is more appropriate. The additional code is to ensure the robustness of the test case and fail in an controlled way.